rules/summary-recording-sprint-reviews/rule.md

```diff
@@ -0,0 +1,71 @@
+---
+type: rule
+title: Do you create a summary and recording at the end of Sprint Reviews for
+  important stakeholders?
+uri: summary-recording-sprint-reviews
+authors:
+  - title: Jack Pettit
+    url: https://ssw.com.au/people/jack-pettit/
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan/
+related:
+  - tech-check
+  - record-a-quick-and-dirty-done-video
+  - rules-to-better-video-recording
+created: 2024-04-15T00:46:35.508Z
+guid: 99674806-7353-4d39-ac9a-219620d4e107
+---
+Creating a comprehensive summary and recording of Sprint Reviews is vital for effective communication and documentation, especially when key stakeholders are unable to attend these meetings. Imagine finishing a Sprint Review without a clear record—stakeholders are left without insights into project progress, leading to potential misalignments and missed opportunities for feedback.
+
+<!--endintro-->
+
+## Why Create a Summary and Recording?
+
+1. **Enhanced Clarity and Transparency:** Provides a clear, accessible update for all stakeholders, including those not present.
+2. **Documentation for Future Reference:** Serves as a historical record of decisions and discussions, valuable for future sprints and project continuity.
+3. **Inclusive Stakeholder Engagement:** Ensures that all stakeholders, regardless of their availability to attend, have access to the same information, promoting inclusiveness and alignment.
```

Comment:
I thought the key reason was to promote internal products and engage with the community?  Should this be mentioned?

Reply:
Added community engagement